src/service/performance-impl.js

```diff
@@ -221,7 +221,7 @@ export class Performance {
    * @private
    */
   setFlushParams_(params) {
-    this.viewer_.setFlushParams(params);
+    this.viewer_.sendMessageCancelUnsent('setFlushParams', params, false);
```

Comment:
I [argued](https://github.com/ampproject/amphtml/pull/6191#discussion_r88509749) for getting rid of the parameter entirely. Always return the promise, cause we generate it anyways.